src/shared/components/palette/base-field/abstract-form-field.component.ts

```diff
@@ -0,0 +1,34 @@
+import { Input } from '@angular/core';
+import { CaseField } from '../../../domain/definition';
+import { AbstractControl, FormControl, FormGroup } from '@angular/forms';
+
+export abstract class AbstractFormFieldComponent {
+
+  @Input()
+  caseField: CaseField;
+
+  @Input()
+  formGroup?: FormGroup;
```

Comment:
is there any case where for a form field `formGroup` is not set?

Reply:
FieldReadComponent is used on case view, search results etc. where form-group and registerControl do not apply, hence optional.